config.py

```diff
@@ -13,7 +13,7 @@
     "HXR":["LFE"]
     }
 device_types = ["ATM", "B4C/Ni MIRROR", "B4C/Si MIRROR", "EXS YAG", "GAS ATTEN", "LASER INCOUPLING", "PPM", "REF LASER", "RTDS", "SCATTER SLIT", "WFS TARGET", "XPIM", "SPECTROMETER", "OTHER"]
-plcs = ["plc-kfe-gatt", "plc-kfe-motion", "plc-kfe-rix-motion", "plc-rixs-optics", "plc-tmo-motion", "plc-tmo-optics", "plc-lfe-optics", "plc-xrt-homs", "plc-txi-hxr-optics"]
+plcs = ["plc-kfe-gatt", "plc-kfe-motion", "plc-kfe-rix-motion", "plc-rixs-optics", "plc-tmo-motion", "plc-tmo-optics", "plc-lfe-gatt", "plc-lfe-motion", "plc-lfe-optics", "plc-lfe-vac", "plc-xrt-homs", "plc-txi-hxr-motion", "plc-txi-hxr-optics", "plc-txi-hxr-vac", "plc-txi-optics", "plc-txi-sxr-motion", "plc-txi-sxr-vac",]
```

Comment:
I chatted with Maggie and we concluded that plcs named `-vac` and also the lfe GEM (`plc-lfe-gatt`) did not need to be (should not be) included in the database.